open_academy/views/course_form.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="view_course_form"  model="ir.ui.view">
+        <field name="name">Form</field>
```

Comment:
Please at the name of view records add model.view.type_of_view for example course.view.form